### ISSUE TITLE
"go get" errors out because of "test-data" submodule

### ISSUE THREAD

**Issue by omerkirk** [oschwald/geoip2-golang#13]
Hi I was using your library previously without issues. However when I tried to update my libraries for a fresh install I got the following error. 

git --git-dir=/Users/omerkirk/Projects/go/src/github.com/oschwald/geoip2-golang/.git submodule update --init --recursive
No submodule mapping found in .gitmodules for path 'test-data'
package github.com/oschwald/geoip2-golang: exit status 1

This is probably simple to fix with manually cloning, still it would be a good idea to fix it here.

**Comment by oschwald** [comment]
Thanks for the report. I am unable to reproduce this with 1.5 on linux/amd64 using a new `$GOPATH`. Could you provide me with specific steps to reproduce the issue?

**Comment by jvehent** [comment]
I'm encountering this as well.

```
$ go version
go version go1.5.1 linux/amd64
```

```
$ go get github.com/oschwald/geoip2-golang
# cd .; git --git-dir=/home/myuser/go/src/github.com/oschwald/geoip2-golang/.git submodule update --init --recursive
No submodule mapping found in .gitmodules for path 'test-data'
package github.com/oschwald/geoip2-golang: exit status 1
```

**Comment by oschwald** [comment]
Very strange. I just tested with a new GOPATH on go1.5.1 linux/amd64, and did not experience the issue. What version of Git are you using? What is the content of `github.com/oschwald/geoip2-golang/.gitmodules`?

**Comment by oschwald** [comment]
Are you using vendoring? This sounds like [this Go issue](https://github.com/golang/go/issues/12573).

**Comment by omerkirk** [comment]
You are right I am using the experimental vendoring. I will try to go get
without it and let you know.

**Comment by kovetskiy** [comment]
@oschwald any progress on this problem?

**Comment by oschwald** [comment]
I am going to close this as it appears the issue is fixed in Go 1.6.
